### PR TITLE
Suncalc feature

### PR DESCRIPTION
**Pull request by andrewxhill**
added calculations for the height of the sun for any point

### CODE REVIEW

**Comment by andrewxhill** [comment]
hey @gfiorav who should own this now?

**Comment by gfiorav** [comment]
Hey @andrewxhill , maybe @rafatower can give this to someone from the Data Services team!

File: scripts-available/CDB_SunCalc.sql [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+-- {
+-- Returns a Timestamp With Timezone from Julian date
+--
+-- }{
+CREATE OR REPLACE FUNCTION _CDB_SunCalc_FromJulian(j NUMERIC) RETURNS timestamptz as $$ 
+DECLARE
+    dayMs NUMERIC = 60 * 60 * 24;
+    J1970 NUMERIC = 2440588;
+    J2000 NUMERIC = 2451545;
```
Comment (pnorman) [inline scripts-available/CDB_SunCalc.sql]: This constant doesn't seem to be used

**Comment by rafatower** [comment]
@andrewxhill @rochoa we need to define workflows for changes in the extension. On one hand we're bloating this extension with diverse functionality. On the other hand we have to automate so that we can have self-service deploys.

File: scripts-available/CDB_SunCalc.sql [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+BEGIN 
+    RETURN _CDB_SunCalc_ToJulian(date) - J2000;
+END; 
+$$ language plpgsql IMMUTABLE;
+-- }
+
+-- {
+-- A set of utlitiy functions for SunCalc calculations
```
Comment (pnorman) [inline scripts-available/CDB_SunCalc.sql]: typo

**Comment by gfiorav** [comment]
I think we can close this now. Feel free to reopen if tests are fixed!

**Pull request closed without merging**